internal/plugins/scorecard/api.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2021 The Operator-SDK Authors
```

Comment:
Would be helpful have a kutll plugin instead of the scorecard one? 
Would not make sense we are able to enable/disable the kutll scaffold when we wish?